ui/partials/main/thRelatedBugQueued.html

```diff
@@ -1,7 +1,7 @@
 <span class="btn-group pinboard-related-bugs-btn">
     <a class="btn btn-xs related-bugs-link"
        title="{{::bug.summary}}"
-       href="https://bugzilla.mozilla.org/show_bug.cgi?id={{::bug.id}}"
+       href="{{bug_url_id1}}"
```

Comment:
for this case, this uses the ``PluginCtrl``, which already injects ``thUrl``.  So you just need to expose the ``thUrl`` function on the scope so you can call it.  If your function in ``thUrl`` is called ``getBugUrl`` then you would add:

    $scope.getBugUrl = thUrl.getBugUrl;

Then you would have this line 4 look like:

    href="{{ getBugUrl(bug.id) }}